Graphana and prometheus fixed (#14)
Volodimirich/MadeFinalProject: README.md

- `http://127.0.0.1:8000/` get hello;
- `http://127.0.0.1:8000/papers` get all papers;
- `http://127.0.0.1:8000/docs` `Swagger UI` with other API methods.

## Monitoring and logging
Prometheus was chosen as the monitoring system. 

Prometheus can be founded on next address - `http://127.0.0.1:9090`

The following command is required to verify access to the database:
`http://127.0.0.1:9090/targets?search=`. After redirecting you should
 check availability of `http://mongodb-exporter:9216/metrics`.

> For monitoring Graphana was chosen. To use it you should do next steps:

Reviewer: ```suggestion
As a dashboard Grafana was chosen. To use it you should do next steps:
```

1. Go to `http://127.0.0.1:3000` (login: _admin_, password: _pass@123_).

> 2. In opened window go to Datasources -> Prometheus. In the new window set next url - `http://prometheus:9090`. After 

Reviewer: Can we somehow add `prometheus` automatically with config?
